wagtail/admin/templates/wagtailadmin/reports/workflow.html

```diff
@@ -0,0 +1,108 @@
+                        </th>
+                        <th>
+                            {% trans 'Status' %}
+                        </th>
+                        <th>
+                            {% trans 'Tasks' %}
+                        </th>
+                        <th>
+                            {% trans 'Requested by' %}
+                        </th>
+                        <th>
+                            {% trans 'Started at' %}
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for workflow_state in object_list %}
+                        <tr>
+                            <td>
+                                <b>{{ workflow_state.workflow }}</b>
+                            </td>
+                            <td>
+                                <a href="{% url 'wagtailadmin_pages:edit' workflow_state.page.id %}">
+                                    {{ workflow_state.page.get_admin_display_title }}
+                                </a>
+                            </td>
+                            <td>
+                                <a href="{% url 'wagtailadmin_pages:workflow_history_detail' workflow_state.page.id workflow_state.id %}" class="status-tag primary">
+                                    {{ workflow_state.get_status_display }}
+                                </a>
+                            </td>
+                            <td>
+                                <h2>{{ workflow_name }}</h2>
+                                {% trans 'Approved task' as approved_title %}
+                                {% trans 'Incomplete task' as incomplete_title %}
+                                {% for task in workflow_state.all_tasks_with_status %}
+                                    <span data-wagtail-tooltip="{{ task.name }}: {{ task.status_display }}">
+                                        {% if task.status == 'approved' %}
+                                            {% wagtail_icon name="success" title=approved_title %}
+                                        {% else %}
+                                            {% wagtail_icon name="radio-empty" title=incomplete_title %}
+                                        {% endif %}
+                                    </span>
+                                {% endfor %}
+                            </td>
+                            <td>{{ workflow_state.requested_by }}</td>
+                            <td>{{ workflow_state.created_at }}</td>
+                        </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+        {% else %}
+            <p>{% trans "No pages have been submitted for moderation yet" %}</p>
+        {% endif %}
+    </div>
+{% endblock %}
+
+{% block extra_js %}
+    {{ block.super }}
+
+    {# load editor js to get tooltips #}
+    {% include "wagtailadmin/pages/_editor_js.html" %}
```

Comment:
Let's not use _editor_js.html as a Random Grab-Bag Of Useful Javascript for things that aren't the page editor, please. Presumably this could just be an import of wagtailadmin/js/vendor/bootstrap-tooltip.js instead...